scripts/ensure-venv.sh

```diff
@@ -2,40 +2,42 @@
 
 # optionally opt out of virtualenv creation
 # WARNING: this will be removed (most likely renamed) soon!
-if [ "$SENTRY_NO_VIRTUALENV_CREATION" == "1" ]; then
+if [[ "$SENTRY_NO_VIRTUALENV_CREATION" == "1" ]]; then
     exit 0
 fi
 
 venv_name=".venv"
-python_bin="python2.7"
+python_version="3.6"
 
-if [ "$SENTRY_PYTHON3" = "1" ]; then
-    venv_name=".venv3"
-    python_bin="python3.6"
+if [[ "$SENTRY_PYTHON2" = "1" ]]; then
+    venv_name=".venv2"
+    python_version="2.7"
 fi
 
-die() {
-    cat >&2 "${@}"
+die () {
+    cat <<EOF
+$@
+EOF
     exit 1
 }
 
-if [ -n "$VIRTUAL_ENV" ]; then
-    # we're enforcing that virtualenv be in .venv, since future tooling e.g. venv-update will rely on this.
-    if [ "$VIRTUAL_ENV" != "${PWD}/${venv_name}" ]; then
-        die "You're in a virtualenv, but it's not in the expected location (${PWD}/${venv_name})"
-    fi
-
-    # TODO: Update this to strictly check .python-version
-    if [ "$SENTRY_PYTHON3" = "1" ]; then
-        python -c "import sys; sys.exit(sys.version_info[:2] != (3, 6))" ||
-            die "For some reason, the virtualenv isn't Python 3.6."
+if [[ -n "$VIRTUAL_ENV" ]]; then
+    major=`python -c "import sys; print(sys.version_info[0])"`
```

Comment:
Could we please keep this fast? Since it's called by direnv now. We're invoking 2 python interpreters and executing python code each time.

Just parse `python -V`. Like this:

    python -V | sed 's/\./ /g' | read _ python_major python_minor python_patch

Reply:
Surprisingly, the variables do not become available for the rest of the script.
I have reversed it for now. If you want to investigate it please go ahead and open a follow up PR.

Reply:
Yeah, I got this. Sorry, the reason is pipes involve subshells, so those variables won't make it to the parent shell.

The only way I can think of right now is to sed into a file to persist that data. Then `read` it in the parent shell. But there's surely a better way?

Reply:
Gonna resolve this and follow up in the future.